src/features/import/hooks/useDateConfig.ts

```diff
@@ -54,7 +54,12 @@ export default function useDateConfig(column: DateColumn, columnIndex: number) {
 
   const personNumberFormats: PersonNumberFormat[] = [];
 
-  if (organization) {
+  if (
+    organization &&
+    (organization.country === 'se' ||
```

Comment:
This won't work, because `country` should generally be uppercase (although there are some outliers which are lowercase). So except for those outliers, this expression will always be false, even for Swedish (`SE`) organizations.